src/lib/blueprint.ts

```diff
@@ -2,6 +2,24 @@ import type { Openapi } from './openapi.js'
 
 export interface Blueprint {
   name: string
+  language: string
+  methodDescription: string
+  versionInformation: string
+  category: string // e.g. Devices, Access Codes, Locks, etc.
+  parameters: Array<{
+    name: string
+    type: string
+    optionality: 'optional' | 'required'
+    description: string
+  }>
+  returnValue: {
+    type: string
+    description: string
+  }
+  examples: Array<{
+    codeSnippet: string
+    explanation: string
+  }>
```

Comment:
Your blueprint type doesn't have a general `routes` or similar property. Instead, method descriptions, parameters, return values, etc., are provided at the top level. Are you proposing to create a blueprint per endpoint? My understanding was that we were including all the routes in the blueprint. Maybe I misunderstood.
@razor-x wdyt?

Reply:
The Blueprint type is the root type, it needs to contain `routes` or `endpoints` etc.